receiver/githubreceiver/README.md

```diff
@@ -55,7 +55,7 @@ receivers:
         initial_delay: 1s
         collection_interval: 60s
         scrapers:
-            github:
+            scraper:
```

Comment:
Why change the name of this scraper? Now that this receiver is named "GitHub receiver" and not "Git Provider receiver", I suppose there will be only one scraper in this receiver, is this correct? If yes, the configuration should not have `scrapers::github` or `scrapers:scraper` at all, but rather the `metrics` configuration option should be a top-level option in the receiver. Does it make sense?

Reply:
Oh, interesting, so you see a use case in defining more than one scraper in a single receiver. I suppose this is OK then. Perhaps we can add more examples in the docs to make it clearer for useres (as a separate contribution). Thanks for explaining this!